src/Controls/src/Core/Shapes/Path.cs

```diff
@@ -1,8 +1,26 @@
+using System;
 using System.ComponentModel;
+using Microsoft.Maui.Graphics;
+
 namespace Microsoft.Maui.Controls.Shapes
 {
-	public class Path : Shape
+	public class Path : Shape, IShape
 	{
+		public Path() : base()
+		{
+		}
+
+		public Path(Geometry data) : this()
+		{
+			Data = data;
+		}
+
+		public PathF PathForBounds(Graphics.Rectangle rect)
+		{
+			return Data.PathForBounds(new Graphics.Rectangle(0, 0, Width, Height))
+				.AsScaledPath((float)Width / (float)rect.Width);
```

Comment:
Again, since the Path has a Width/Height, we have to assume the path is for a given set of bounds, and we scale it for the given rectangle.